app/src/main/java/com/gathering/android/event/myevent/addevent/invitation/AttendeeListAdapter.kt

```diff
@@ -0,0 +1,70 @@
+package com.gathering.android.event.myevent.addevent.invitation
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.gathering.android.common.ImageLoader
+import com.gathering.android.databinding.ItemAttendeeBinding
+import dagger.hilt.android.qualifiers.ApplicationContext
+import javax.inject.Inject
+
+class AttendeeListAdapter @Inject constructor(
+    @ApplicationContext private val context: Context,
+    private val imageLoader: ImageLoader
+) : RecyclerView.Adapter<AttendeeListAdapter.ViewHolder>() {
+
+    private var attendeeItemList: MutableList<String> = mutableListOf()
+    private var onAttendeeRemoveClicked: (attendee: String) -> Unit = {}
+    private val li = LayoutInflater.from(context)
+
+    @SuppressLint("NotifyDataSetChanged")
+    fun addAttendeeItem(attendee: String) {
+        this.attendeeItemList.add(attendee)
+        notifyDataSetChanged()
+    }
+
+    @SuppressLint("NotifyDataSetChanged")
```

Comment:
I believe notifyDataSetChanged is used to add or remove from the list? if this is the case,  as far as I know, "notifyDataSetChanged" will refresh the whole recyclerView even if there is only one item in the list added or removed. in that case, what happens when we have a larger list?

Reply:
Good point, You are  right I should change it to notifyItemInserted and notifyItemRemoved